config-examples/config.example.nats-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
don't need as this is for internal use only, I don't want it to confuse customers